specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/analyzeconversations.json

```diff
@@ -122,11 +128,11 @@
       "type": "object",
       "description": "This is the parameter set of either the conversation application itself or one of the target projects.",
       "required": [
-        "projectType"
+        "targetType"
       ],
-      "discriminator": "projectType",
+      "discriminator": "targetType",
       "properties": {
-        "projectType": {
+        "targetType": {
```

Comment:
This doesn't seem to match the description below, which states, "The type of the project." And given that we're trying to unify on the concepts of "project", is this appropriate?

/cc @rokulka

Reply:
@ChongTang  - I see that you've updated to 'targetType' , sounds better than 'projectType'.  
@heaths  - you were mentioning to change suffixes to 'Kind' instead of 'Type' , would that apply here?

Reply:
Ah, yeah, good point. I need to remember that for next time (it's a recent request from some SDK archboard members).